### PR TITLE
File Upload - Visitor can upload more than 25 files after removing of last uploaded file

### PR DESCRIPTION
**Pull request by kuldar-daniel**
This PR fixes issue, where removing the file would remove all of the uploads where the local file is the same, which messes up the file count. Add UUID to each individual upload for them to be different.

### CODE REVIEW

**Review by dukhovnyi — COMMENTED** [review]
Sorry, I didn't catch how local file comparison affects file count limitation.
Perhaps we need to analyze the mechanism that counts the allowed file amount and rework it.
And in sense of adding unit-tests, I'd like to propose adding tests for this issue also.

**Comment by kuldar-daniel** [comment]
> Sorry, I didn't catch how local file comparison affects file count limitation. Perhaps we need to analyze the mechanism that counts the allowed file amount and rework it. And in sense of adding unit-tests, I'd like to propose adding tests for this issue also.

`uploads.removeAll(where: { $0 == upload })`

when tapping close, all of the ones with same local file would be removed, but the UI would still display them.

File: GliaWidgets/Lib/Upload/FileUpload.swift [hunk 1]
```diff
@@ -53,6 +55,7 @@ class FileUpload {
 
     private let storage: DataStorage
     private let environment: Environment
+    private let uuid = UUID()
```
Comment (igorkravchenko) [inline GliaWidgets/Lib/Upload/FileUpload.swift:58]: Let's use DI for bringing generation of UUID here.

**Review by igorkravchenko — COMMENTED** [review]
I support @dukhovnyi on this, that it looks incorrect to assume equitability based only on UUID and ignoring other fields, even if it solves current issue for now. Also it will be beneficial if we add unit test(s) that will prove that this issue is no longer exists.

**Comment by kuldar-daniel** [comment]
Added test + removed the fileupload equatable protocol alltogether

**Review by igorkravchenko — APPROVED** [review]
Great job! Looks good 👍

File: GliaWidgetsTests/Lib/Upload/FileUploader.Environment.Failing.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@testable import GliaWidgets
+extension FileUploader.Environment {
```
Comment (dukhovnyi) [inline GliaWidgetsTests/Lib/Upload/FileUploader.Environment.Failing.swift:2]:
```suggestion

extension FileUploader.Environment {
```

**Review by dukhovnyi — APPROVED** [review]

**Pull request merged**